### PR TITLE
[IOTDB-6239] Show regions display error create time

### PR DESCRIPTION
**Pull request by CRZbulabula**
**Jira:** https://issues.apache.org/jira/browse/IOTDB-6239

**Bug fix test:** the create time is correct in this PR.
bug-fix

**Compatibility test:** the ConfigNode is still compatible with earlier version, through the create time of Region can't be fixed to correct.
compatible

### CODE REVIEW

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java [hunk 1]
```diff
@@ -39,40 +40,48 @@
...
 import java.util.stream.Collectors;
 
-/** Create regions for specific StorageGroups. */
+/** Create regions for specified Databases. */
 public class CreateRegionGroupsPlan extends ConfigPhysicalPlan {
 
-  // Map<StorageGroupName, List<TRegionReplicaSet>>
+  // Map<Database, List<TRegionReplicaSet>>
   protected final Map<String, List<TRegionReplicaSet>> regionGroupMap;
+  protected final Map<TConsensusGroupId, Long> regionGroupCreateTimeMap;
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:48]: Maybe you don't need to maintain a map, just a value? A batch of RegionGroups can use the same time？

Comment (OneSizeFitsQuorum) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:48]: Maybe you don't need to maintain a map, just a value? A batch of RegionGroups can use the same time？

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java [hunk 2]
```diff
@@ -96,28 +105,46 @@ protected void serializeImpl(DataOutputStream stream) throws IOException {
...
       int regionReplicaSetNum = buffer.getInt();
       for (int j = 0; j < regionReplicaSetNum; j++) {
         TRegionReplicaSet regionReplicaSet =
             ThriftCommonsSerDeUtils.deserializeTRegionReplicaSet(buffer);
-        regionGroupMap.get(storageGroup).add(regionReplicaSet);
+        regionGroupMap.get(database).add(regionReplicaSet);
+      }
+    }
+
+    if (buffer.hasRemaining()) {
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:141]: We need to consider setting a default value (maybe the current time) when restarts with no data.

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java (hunk 1 above)
Comment (CRZbulabula) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java]: Fixed!!!!!!!!

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java (hunk 2 above)
Comment (CRZbulabula) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:136]: Fixed

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java [hunk 3]
```diff
@@ -59,20 +61,28 @@ public Map<String, List<TRegionReplicaSet>> getRegionGroupMap() {
     return regionGroupMap;
   }
 
-  public void addRegionGroup(String storageGroup, TRegionReplicaSet regionReplicaSet) {
+  public long getCreateTime() {
+    return createTime;
+  }
+
+  public void setCreateTime(long createTime) {
```
Comment (wangchao316) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:68]: setCreateTime only for test?

Comment (CRZbulabula) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/region/CreateRegionGroupsPlan.java:68]: No. This interface is also invoked at CreateRegionGroupsProcedure, where set the RegionGroup creation time.

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**